cli/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
I agree with the two-spaces approach, but I wouldn't make this a **root** `.editorconfig`. I can imagine a project where both frontend and backend standards are at play and they contradict each other. In the .NET world, for example, it is pretty standard to have 4 spaces or tabs.

So, I think we should keep the `.editorconfig` with the 2 spaces, but not make it a root. As such, it would only affect what is generated with this seed (which is intended) and would not clash with any other standard in the grand-scheme of other projects (which I believe is desirable).